src/components/MenuBar/MenuBar.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import MenuIcon from '@material-ui/icons/Menu';
+import IconButton from '@material-ui/core/IconButton';
+import useTheme from '@material-ui/core/styles/useTheme';
+import Collapse from '@material-ui/core/Collapse';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import Typography from '@material-ui/core/Typography';
+import { navigate } from 'gatsby';
+
+import { makeStyles } from '@material-ui/core/styles';
+
+import ButtonBar from './ButtonBar';
+import { ReactComponent as HackLogo } from '../../images/hack-logo.svg';
+
+const useStyles = makeStyles(theme => {
+	/**
+	 * This is the limit of the screensize where the MenuBar
+	 * should switch between desktop and mobile.
+	 */
+	const menuBarAdaptiveThreshold = theme.breakpoints.values.sm * 1.3;
+	return {
+		logohome: {
+			display: 'flex',
+			alignItems: 'center',
+			justifyContent: 'center',
+			cursor: 'pointer'
+		},
+		toolbar: {
+			backgroundColor: 'white',
+			justifyContent: 'space-between',
+			padding: '0% 6%'
+		},
+		logo: {
+			height: '1.8rem',
+			marginRight: theme.spacing(1.5)
+		},
+		wordmark: {
+			color: 'black',
+			fontWeight: 700,
+			fontSize: '1.5em',
+			fontFamily: theme.typography.fontFamily
+		},
+		menubtn: {
+			// desktop
+			display: 'none',
+			[theme.breakpoints.down(menuBarAdaptiveThreshold)]: {
+				// mobile
+				display: 'inline'
+			}
+		},
+		desktopMenuBar: {
+			// desktop
+			display: 'inline',
+			[theme.breakpoints.down(menuBarAdaptiveThreshold)]: {
+				// mobile
+				display: 'none'
+			}
+		},
+		mobileMenuBar: {
+			// desktop
+			display: 'none',
+			[theme.breakpoints.down(menuBarAdaptiveThreshold)]: {
+				// mobile
+				display: 'inline'
+			},
+			// default styling
+			boxShadow: theme.shadows[4],
+			position: 'fixed',
+			zIndex: '1',
+			overflow: 'hidden',
+			backgroundColor: theme.palette.grey[100],
+			width: '100%'
+		},
+		mobileBtnContainer: {
+			display: 'flex',
+			flexDirection: 'column',
+			padding: '3%'
+		}
+	};
+});
+
+function MenuBar() {
+	const classes = useStyles();
+	const theme = useTheme();
+	const [menuOpen, setMenuOpen] = useState(false);
+	const toggleMenu = () => setMenuOpen(open => !open);
+	const isMobile = useMediaQuery(theme.breakpoints.down(theme.breakpoints.values.sm * 1.3));
+
+	const desktopMenuBar =
+		<AppBar position="sticky">
+			<Toolbar className={classes.toolbar}>
+				<div
```

Comment:
```suggestion
				<nav
```

Reply:
Should just the logo and wordmark be enclosed in `nav` tags? Or should the `ButtonBar` also be enclosed in `nav` tags rather than `div`?